### DISCUSSION TITLE
Shows build for one device issue

### DISCUSSION THREAD

**Discussion by Prashant-1695** [ROM-builders/temporary#9780] [Q&A]
https://cirrus-ci.com/task/6259623207370752

Only doing for one device, still it shows like this

**Comment by Apon77** [comment]
> https://cirrus-ci.com/task/6259623207370752
> 
> Only doing for one device, still it shows like this

use lunch or brunch, only one command like this, https://github.com/ROM-builders/temporary/blob/mido-crdroidandroid-12.1-Apon77/build_rom.sh#L6-L9
